pilot/pkg/kube/inject/webhook_test.go

```diff
@@ -863,12 +866,35 @@ func loadConfigMapWithHelm(t *testing.T) string {
 	return body.Template
 }
 
-func getHelmValues(t *testing.T) string {
+func getHelmValues(t testing.TB) string {
 	t.Helper()
 	valuesFile := filepath.Join(helmChartDirectory, helmValuesFile)
 	return string(util.ReadFile(valuesFile, t))
 }
 
+func mergeParamsIntoHelmValues(params *Params, vals string, t testing.TB) string {
+	t.Helper()
+	if params == nil {
```

Comment:
Optional, Instead of nil, an empty params can be the same? I think that can be easier to reason about.